components/Editor.js

```diff
@@ -172,6 +174,14 @@ class Editor extends React.Component {
                   .replace(/%23/g, '#')
                   .replace(/%0A/g, '\n')
               )
+              // remove other fonts which are not used
+              .then(
+                dataUrl =>
+                  (dataUrl = dataUrl.replace(
+                    new RegExp('@font-face { font-family: (?!"*' + fontFamily + ').*?}s?', 'g'),
```

Comment:
What is the `s?` part of the regex for?

Reply:
\s matches a space, a tab, a line break, or a form feed.

Reply:
I think we won't be needing that.

Reply:
It's just in cases in the future the output is different, in which case `\s` is safer. To be fair, I don't think it will ever change, especially b/c `dom-to-image` hasn't gotten released in months.

Reply:
Also, don't be sorry 🙂 we are working on this together!